<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateInvestimento</class>
 <widget class="QWidget" name="CreateInvestimento">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>249</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>289</width>
    <height>249</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>289</width>
    <height>249</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Investimentos</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="12" column="1" alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="cancelButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" alignment="Qt::AlignLeft|Qt::AlignVCenter">
      <widget class="QLabel" name="labelValor">
       <property name="text">
        <string>Valor:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" alignment="Qt::AlignLeft|Qt::AlignVCenter">
      <widget class="QLabel" name="labelData">
       <property name="text">
        <string>Data:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0" alignment="Qt::AlignLeft|Qt::AlignVCenter">
      <widget class="QLabel" name="labelDesc">
       <property name="text">
        <string>Local:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0" alignment="Qt::AlignRight">
      <widget class="QPushButton" name="saveButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Salvar</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="QLineEdit" name="valorInput">
       <property name="placeholderText">
        <string>200.00</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="2">
      <widget class="QLineEdit" name="localInput"/>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QDateEdit" name="dateInput">
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="date">
        <date>
         <year>2000</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
